Automattic/wp-calypso
# Add shadow for preview and checklist hover color

#### Proposed Changes

* This PR updates CSS shadow styling for the site preview iframe, progress bar related CSS. and checklist item hover CSS.

#### Testing Instructions



* Checkout PR
* Navigate to: http://calypso.localhost:3000/setup/launchpad?flow=newsletter&siteSlug=agrullon95.wordpress.com
* Ensure that the font color of a checklist item changes to blue if you hover over it and it's not disabled.
* Compare iframe shadow styling and progress bar to Figma designs: 9wpoYJrMlYKAKUYvVBMN74-fi-150%3A3809

#### Pre-merge Checklist






Related to #66575
